CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased 0.75.0
+## Unreleased 0.74.4
```

Comment:
Not sure why the change here.

Reply:
I wasn't sure, but seeing as you released the 0.74.3 tag yesterday, I thought we had to bump it, I will revert it.